detekt.yml

```diff
@@ -132,7 +132,7 @@ complexity:
   LongParameterList:
     active: true
     functionThreshold: 6
-    constructorThreshold: 7
+    constructorThreshold: 8
```

Comment:
Evidence to improve interaction with data layer. Could be more concise. Subsequent PR's to keep this small